inc/user.class.php

```diff
@@ -126,7 +126,7 @@ function showForm($ID) {
 
       echo "<tr class='tab_bg_1'>";
       echo "<td><label>";
-      echo __("Enable filtering on the groups assignment", "escalade");
+      echo __("Bypass filtering on the groups assignment", "escalade");
```

Comment:
This reverse the option and mean that after the update every user will have it's settings inverted.

Maybe this could be avoided if the database was updated during the plugin upgrade to match the new behavior or this field ?